frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -351,6 +351,33 @@ public void testNoopPrediction() throws InterruptedException {
         testPredictions("noop", "OK", null);
     }
 
+    @Test(
+            alwaysRun = true,
+            dependsOnMethods = {"testSetDefaultVersionNoop"})
+    public void testLoadModelWithNakedDirModelArchive() throws InterruptedException {
```

Comment:
@harshbafna Why are these other files included in this PR? It should only contain torch package version related items. Please split into separate PRs if you are adding additional tests for NakedMAR